### DISCUSSION TITLE
Brief Notes for Building on Windows

### DISCUSSION THREAD

**Discussion by aa5sh** [foldynl/QLog#504] [General]
@foldynl here are some brief notes I used on building for windows.  I know that part of the Readme does not have much detail.  I have used these on two workstations and they worked ok.

[QLog Build on Windows Notes.txt](https://github.com/user-attachments/files/17791657/QLog.Build.on.Windows.Notes.txt)

**Comment by foldynl** [comment]
Many thanks. Do you agree to add it to README and WIKI ?

I can see that you are using QT 6.8. Unfortunately, I tried it, and QLog will need some changes related to the 6.8. First, it requires VC 2022 (currently using VC2019) but more importantly, there’s an issue with the Time formats and time zones. I’ll try to fix it, but I won’t be able to get it into version 0.40.

**Comment by aa5sh** [comment]
Yeah I'm ok with adding it.

On this machine I just had vs_code installed.  When I went to download VS
2019 looked like Microsoft had removed the installer for the community
version of it, so I had to install 2022.  With that I installed QT 5.15.2
and 6.8, but with 5.15.2 I kept getting an error about missing qtwebengine
I think it was.  I will try again now that I know I can get it to build on
Windows.

**Comment by aa5sh** [comment]
Ok I did find a link to vs2019 from a Microsoft URL:  https://c2rsetup.officeapps.live.com/c2r/downloadVS.aspx?sku=community&channel=Release&source=VSLandingPage&version=VS2019&cid=2030

And I installed 5.15.2 and was able to build QLog with 5.15.2

The only problem I encountered was with OmniRig it was giving me a meesage saying exception was not declared in `connect(omniRigInterface, &OmniRig::OmniRigX::exception, this, &OmnirigRigDrv::COMException);`
for both OmnirgRigDrv.cpp and Omnirigv2RigDrv.cpp .  I just commented those out for testing.  But I am not sure why that definition wasn't made.

**Comment by foldynl** [comment]
I think there is no need to somehow tune it for 2019. I can use VC 2019 with QT 6.6. However, I hope that 0.40 will be the last version with QT 6.6 and 0.41 will be with QT 6.8. Unfortunately, VC 2022 is needed for 6.8, so I have to upgrade my build environment. So I would leave the mention of VC 2022.
